painttiming.bs

```diff
@@ -179,8 +184,8 @@ The {{PerformancePaintTiming}} interface {#sec-PerformancePaintTiming}
 * The {{PerformanceEntry/startTime}} attribute's getter must return a {{DOMHighResTimeStamp}} of when the paint occured.
 * The {{PerformanceEntry/duration}} attribute's getter must return 0.
 
-NOTE: A user agent implementing {{PerformancePaintTiming}} would need to include <code>"paint"</code> in {{PerformanceObserver/supportedEntryTypes}} for {{Window}} contexts.
-This allows developers to detect support for paint timing.
+NOTE: A user agent implementing {{PerformancePaintTiming}} would need to include <code>"paint"</code> in {{PerformanceObserver/supportedEntryTypes}} for any [=top level browsing context=], and may include it for any [=nested browsing context=].
+    This allows developers to detect support for paint timing for a particular [=browsing context=].
 
```

Comment:
How is this @yoavweiss ?

Reply:
Better, but this just handles the feature detection part, not actually making the support optional. Might be better to make the optional support normative, and then just make it clear in the note that "paint" should be included only in global objects which trigger paint timing entries.

Reply:
OK, revised.

Reply:
See revised version in "Mark paint timing"